features/product/viewing_products/viewing_product.feature

```diff
@@ -0,0 +1,15 @@
+@viewing_products
+Feature: Viewing a product details
+    In order to see products detailed information
+    As a visitor
+    I want to be able to browsing products
+
+    Background:
+        Given the store operates on a channel named "Web Channel"
```

Comment:
Why not using France? Would simplify the background, because currency is irrelevant for this scenario/feature.